File: Java/1808 - Hamming Distance.java
import java.util.*;

public class COJ2 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		boolean continuar = true;

		while (continuar) {
			String s = sc.nextLine();

			if (s.equalsIgnoreCase("X")) {
				continuar = false;
			} else {

				int suma = 0;
				String st = sc.nextLine();

				for (int i = 0; i < s.length(); i++) {

					if (s.charAt(i) != st.charAt(i)) {
						suma++;
					}

				}

				System.out.print("Hamming distance is " + suma + ".\n");
			}
		}
	}
}

File: c++/1808 - Hamming Distance.cpp
#include <iostream>

#define FOR(a, b, c, d) for (a = b; a < c; a += d)

using namespace std;

int main()
{
	string c1, c2;
	int i, dis;
	while (cin >> c1)
	{
		if (c1 != "X")
		{
			cin >> c2;
			dis = 0;
			FOR(i, 0, c1.length(), 1)
			{
				if (c1[i] != c2[i])
					dis++;
			}
		}
		else
			break;
		cout << "Hamming distance is " << dis << "." << endl;
	}
	return 0;
}
